docs/jobs.md

```diff
@@ -0,0 +1,477 @@
+Cozy Jobs
+=========
+
+Jobs are designed to represent asynchronous tasks that your cozy can execute. These tasks can be scheduled in advance, recurring or sudden and provide various services.
+
+At the time, we do not provide "generic" and programmable tasks. This list of available workers is part of the defined API.
+
+The job queue can be either local to the cozy-stack when used as a monolithic instance (self-hosted for instance) or distributed via a redis server for distributed infrastructures.
+
+This doc introduces two cozy types:
+
+  - `io.cozy.jobs` for jobs
+  - `io.cozy.triggers` for triggers
+
+
+Triggers
+--------
+
+Jobs can be launched by three different types of triggers:
+
+  - `@cron` to schedule recurring jobs scheduled at specific times
+  - `@interval` to schedule periodic jobs executed at a given fix interval
+  - `@event` to launch a job after a change in the cozy
```

Comment:
This week-end, I thought of a new type of trigger: `@webhook`. It would create an unguessable URL that we can give to external services for pushing a job. For example, we could give a webhook URL to github to update on our cozy the app we are working on after each git push.